clients/runtime/src/rpc.rs

```diff
@@ -1436,17 +1436,24 @@ impl ReplacePallet for SpacewalkParachain {
 
 #[async_trait]
 pub trait StellarRelayPallet {
-	async fn is_public_network(&self) -> Result<bool, Error>;
+	async fn is_public_network(&self) -> bool;
```

Comment:
addresses https://github.com/pendulum-chain/spacewalk/blob/bda384bcc6db3da88198c714851a261dcdea1063/clients/vault/src/system.rs#L826

By default we're assuming `public_network` = true anyway:
https://github.com/pendulum-chain/spacewalk/blob/bda384bcc6db3da88198c714851a261dcdea1063/clients/vault/src/system.rs#L713-L714 So might as well have this return a bool.

This function will be called multiple times in `system.rs`.
Updating the faucet itself will need to check the stellar network.